### PR TITLE
Add support for sub in JatsContentBundle

### PR DESCRIPTION
**Pull request by nlisgo**
Improves: https://github.com/libero/libero/issues/106

Blocked by: https://github.com/libero/pattern-library/pull/51

### CODE REVIEW

**Comment by nlisgo** [comment]
@thewilkybarkid you mentioned that we can hold off on advancing the libero schema for now. Does that mean that we don't need to add a `SubVisitor` to the `LiberoContentBundle` at this moment?

**Comment by thewilkybarkid** [comment]
@nlisgo Could you try out #44? The Sub pattern came first, so we can avoid having the Sub update here.

File: vendor-extra/JatsContentBundle/src/Resources/config/services.xml [hunk 1]
```diff
@@ -100,6 +100,12 @@
             <tag name="libero.view_converter"/>
         </service>
 
+        <service id="Libero\JatsContentBundle\ViewConverter\SubVisitor"
+                 class="Libero\JatsContentBundle\ViewConverter\SubVisitor">
```
Comment (thewilkybarkid) [inline vendor-extra/JatsContentBundle/src/Resources/config/services.xml]:
```suggestion
            class="Libero\JatsContentBundle\ViewConverter\SubVisitor">
```

File: vendor-extra/JatsContentBundle/tests/ViewConverter/SubVisitorTest.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+     * @test
+     */
+    public function it_sets_the_template_and_text_argument() : void
+    {
+        $visitor = new SubVisitor($this->createDumpingConverter());
+
+        $element = $this->loadElement(
+            <<<XML
+<jats:sub xmlns:jats="http://jats.nlm.nih.gov">
+    foo <jats:bold>bar</jats:bold> baz
```
Comment (thewilkybarkid) [inline vendor-extra/JatsContentBundle/tests/ViewConverter/SubVisitorTest.php]: While we're not allowing mixed content in the spec, right to leave it working here (as we're allowing it to be changed).

**Pull request merged**